### ISSUE TITLE
Details about training: learning rate

### ISSUE THREAD

**Issue by Z1740220020** [mrnabati/CenterFusion#52]
Hello author, I appreciate your wonderful work. 

But when I used your CenterFusionE60 pretraining model and continued training with the default train.py file, I got very poor results. Even with just 2 epochs of training, the model's performance declined rapidly.
 How is the author trained? What are the training details? Is my learning rate setting wrong?

**Comment by harborsarah** [comment]
> Hello author, I appreciate your wonderful work.
> 
> But when I used your CenterFusionE60 pretraining model and continued training with the default train.py file, I got very poor results. Even with just 2 epochs of training, the model's performance declined rapidly. How is the author trained? What are the training details? Is my learning rate setting wrong?

Hi, did you solve the problem? I have the same issue.
